tests/Unit/SymfonyCache/TagSubscriberTest.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace FOS\HttpCache\Tests\Unit\SymfonyCache;
+
+use Symfony\Component\HttpFoundation\Request;
+use Prophecy\Argument;
+use FOS\HttpCache\SymfonyCache\TagSubscriber;
+use Symfony\Component\HttpFoundation\Response;
+
+class TagSubscriberTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var TagManagerInterface
+     */
+    private $tagManager;
+
+    /**
+     * @var CacheEvent
+     */
+    private $event;
+
+    public function setUp()
+    {
+        $this->tagManager = $this->prophesize('FOS\HttpCache\SymfonyCache\Tag\ManagerInterface');
```

Comment:
Yeah sure.

On Sun, Nov 29, 2015 at 11:46:32PM -0800, David de Boer wrote:
>    In [1]tests/Unit/SymfonyCache/TagSubscriberTest.php:
> 
>  > +
>  > +class TagSubscriberTest extends \PHPUnit_Framework_TestCase
>  > +{
>  > +    /**
>  > +     * @var TagManagerInterface
>  > +     */
>  > +    private $tagManager;
>  > +
>  > +    /**
>  > +     * @var CacheEvent
>  > +     */
>  > +    private $event;
>  > +
>  > +    public function setUp()
>  > +    {
>  > +        $this->tagManager = $this->prophesize('FOS\HttpCache\SymfonyCache\Tag\ManagerInterface');
> 
>    Would you be okay with changing this to Mockery? That’s what we now use in
>    this library’s test suites. It may be a bit confusing to have different
>    ways of mocking.
> 
>    —
>    Reply to this email directly or [2]view it on GitHub.
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/FriendsOfSymfony/FOSHttpCache/pull/253#discussion_r46115125
>    2. https://github.com/FriendsOfSymfony/FOSHttpCache/pull/253/files#r46115125
>    3. https://github.com/FriendsOfSymfony/FOSHttpCache/pull/253/files#r46115125